### ISSUE TITLE
Creating multiple wg tunnels

### ISSUE THREAD

**Issue by GeneriumMadness** [angristan/wireguard-install#441]
It would be very nice to add the function of creating and maintaining several vpn tunnels at once, for example wg0, wg1. The script ask for creating another tunnel as the fifth option of the menu and if there is more than one tunnel choose it right after starting script.

**Comment by ExtremeDot** [comment]
good idea, im trying to make new script based on angrsitan script. let you know when finished

**Comment by gch1p** [comment]
As a temporal hack you can use this branch in my fork: https://github.com/gch1p/wireguard-install/tree/wg-instance

Run script with `WG_INSTANCE` environment variable to create or manage another wireguard server on the same machine.

**Comment by Wesley-Sinde** [comment]
if i already have the original wireuard, and now start using https://github.com/gch1p/wireguard-install/tree/wg-instance in the same server will this affect the already existing wgo instance users?
